.travis.yml

```diff
@@ -39,6 +39,8 @@ install:
   # Install protoc
   - wget https://github.com/google/protobuf/releases/download/v3.6.0/protoc-3.6.0-linux-x86_64.zip -O /travis-install/protoc.zip
   - sudo unzip /travis-install/protoc.zip -d /usr
+  # Install pytorch cpu only (no wheels on pip)
+  - conda install -c pytorch pytorch-cpu
```

Comment:
FYI, it would be better to add these to `tax-requirements.txt` so people can also run the tests locally. Does that work if you do it or do you need to get PyTorch through Conda?

Reply:
There is no CPU only torch wheels on PyPI, so we can insert torch in tox requirement and ask people to download ~500MB

Reply:
However it is possible to download a specific version from pytorch server: e.g.
http://download.pytorch.org/whl/cpu/torch-0.4.1-cp36-cp36m-linux_x86_64.whl